esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/get_esmvaltool/bin/get_esmvaltool.sh

```diff
@@ -0,0 +1,14 @@
+# Remove the ESMValTool and ESMValCore directories, if they exist.
```

Comment:
#!/bin/bash
# Send the output from 'set -x' to 'stdout' rather than 'stderr'.
BASH_XTRACEFD=1
set -eux

needs adding to the top of the .sh file so that it is identified as a bash script and the output is written to the correct file